servers/rendering/rasterizer_rd/shaders/SCsub

```diff
@@ -22,3 +22,9 @@ if "RD_GLSL" in env["BUILDERS"]:
     env.RD_GLSL("ssao_minify.glsl")
     env.RD_GLSL("ssao_blur.glsl")
     env.RD_GLSL("roughness_limiter.glsl")
+    env.RD_GLSL("screen_space_reflection.glsl")
+    env.RD_GLSL("screen_space_reflection_filter.glsl")
+    env.RD_GLSL("screen_space_reflection_scale.glsl")
+    env.RD_GLSL("specular_merge.glsl")
+    
+    
```

Comment:
CI is complaining about these two lines.

Reply:
the pre commit hook that executes black is broken, I had to disable it. I was getting this error all the time and was unable to commit:
```
Do you want to apply that patch (Y - Apply, N - Do not apply, S - Apply and stage files)? [Y/N/S] Y
error: patch failed: servers/rendering/rasterizer_rd/shaders/SCsub:24
error: servers/rendering/rasterizer_rd/shaders/SCsub: patch does not apply
```

Reply:
I'll check that.

Reply:
Seems that `black --diff` generates a diff which is not always a valid patch: https://github.com/psf/black/issues/526
I'll check how to work it around :|